### PR TITLE
Add test coverage for kubelet.container.runtime

### PR DESCRIPTION
**Pull request by greghaynes**
Adding some very simple unit test coverage

**What this PR does / why we need it**: Additional unit test coverage for file with very little coverage

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by spiffxp** [comment]
@k8s-bot ok to test

File: pkg/kubelet/container/runtime_test.go
Comment (k82cn) [inline pkg/kubelet/container/runtime_test.go]: s/2015/2017

File: pkg/kubelet/container/runtime_test.go
Comment (k82cn) [inline pkg/kubelet/container/runtime_test.go]: thanks very much for your effort :). Please use `for loops` for tests, refer to https://github.com/kubernetes/kubernetes/pull/39568/files#diff-5a979eff174acb57bf1b20dc522651e2R3491 for detail.

**Comment by greghaynes** [comment]
Thanks a lot for the fast feedback! sorry I couldn't fix it up as quickly :)

I've updated the PR to address the comments.

**Comment by resouer** [comment]
kindly ping @vishh for approve

**Review by resouer — APPROVED** [review]
Change make sense to me

**Comment by euank** [comment]
This is ridiculous.

A PR shouldn't just *sit* for 3 months for absolutely no reason. There's no objections or pending discussion. There's a LGTM label.
This PR is a failure of the approval and merge process.

Someone once said you catch more contributors with love, attention, and guidance than you do with neglect and botspam.

cc @vishh @Random-Liu as possible approvers.

**Comment by cblecker** [comment]
cc @kubernetes/sig-node-pr-reviews 
/test pull-kubernetes-bazel-build
/test pull-kubernetes-bazel-test
/test pull-kubernetes-e2e-gce

**Comment by resouer** [comment]
Just posted into sig-node slack for more help. This may also need a final review since last round is quite a long time ago.

**Comment by spiffxp** [comment]
The failure of `pull-kubernetes-e2e-gce-etcd3` can safely be ignored.  This PR is ready for merge pending an `/approve` @kubernetes/sig-node-pr-reviews.

The `pull-kubernetes-e2e-gce-etcd3` job got renamed to `pull-kubernetes-e2e-gce` (which is passing for this PR) so no amount of /retesting is going to even trigger it, let alone cause it to pass.

/assign @vishh @Random-Liu 
per the above suggestion for approvers

**Comment by spiffxp** [comment]
ping @vishh @Random-Liu this is ready for approval

**Comment by greghaynes** [comment]
@cblecker I may be missing something, but I thought @spiffxp mentioned above that pull-kubernetes-e2e-gce-etcd3 would never pass due to being renamed, but pull-kubernetes-e2e-gce is passing (which is the new name) therefore this fail can be ignored?

**Comment by BenTheElder** [comment]
Yep this PR just needs approval.

**Pull request closed without merging**
